corehq/form_processor/models.py

```diff
@@ -755,6 +739,10 @@ def get_obj_by_id(cls, case_id):
         from corehq.form_processor.backends.sql.dbaccessors import CaseAccessorSQL
         return CaseAccessorSQL.get_case(case_id)
 
+    @property
+    def default_parent_identifier(self):
+        return CommCareCaseIndexSQL.PARENT_IDENTIFIER
```

Comment:
This doesn't really seem necessary since in both cases the value of the constant is `parent`